Extensions/TweenBehavior/tweentools.ts

```diff
@@ -158,6 +161,26 @@ namespace gdjs {
         getShiftyScene(runtimeScene).add(tween);
       };
 
+      export const tweenVariableNumber2 = (
```

Comment:
maybe add a small comment above this function explaining why we use this one now and not the previously deprecated one?

Reply:
👍

Reply:
I added a detailed comment on the deprecated function